src/modules/pos/validator/components/StakeForm/StakeRow.js

```diff
@@ -71,6 +72,9 @@ const StakeRow = ({
       <span className={`${styles.oldAmountColumn} ${styles.centerContent}`}>
         {!!confirmed && <TokenAmount val={confirmed} token={token} />}
       </span>
+      <span className={`${styles.newAmountColumn} ${styles.centerContent}`}>
```

Comment:
why `newAmountColumn` styles class used for `commission` probably the naming is not correct? or are we reusing some class?